protonmail/srp.go

```diff
@@ -23,10 +20,10 @@ func decodeModulus(msg string) ([]byte, error) {
 	}
 
 	// TODO: check signature key
-	_, err := openpgp.CheckDetachedSignature(nil, bytes.NewReader(block.Plaintext), block.ArmoredSignature.Body)
-	if err != nil && err != openpgperrors.ErrUnknownIssuer {
-		return nil, err
-	}
+	// _, err := openpgp.CheckDetachedSignature(nil, bytes.NewReader(block.Plaintext), block.ArmoredSignature.Body)
```

Comment:
Hmm, I'm a little bit sad to see this go away. What is the exact error you're hitting?

Reply:
@emersion look the second commit